sgx/test/src/kmyth_sgx_test.c

```diff
@@ -184,6 +185,52 @@ void test_unseal_and_export(void)
   return;
 }
 
+void test_seal_unseal_nkl(void)
+{
+  const char *data = "Test of the NKL seal and unseal";
+  size_t data_len = 30;
+  uint8_t *sgx_seal = NULL;
+  size_t sgx_seal_len = 0;
+  uint64_t handle;
+  uint16_t key_policy = SGX_KEYPOLICY_MRSIGNER;
+  sgx_attributes_t attribute_mask;
+
+  attribute_mask.flags = 0;
+  attribute_mask.xfrm = 0;
+
+  uint8_t *cipher_data_decrypted = NULL;
+
+  int sgx_ret_int;
+  uint32_t sgx_ret_uint32_t;
```

Comment:
I don't see sgx_ret_uint32_t being used anywhere. Also, the "_t" suffix (also used with the variable sgx_ret_size_t) usually indicates a type in C. It is not a good idea to use it for variable names. Github's color scheme is even confused.